.github/ISSUE_TEMPLATE/feature_request.yml

```
@@ -0,0 +1,27 @@
+name: Feature request
+description: Suggest an idea for this project
+title: '[Feature request]: '
+labels:
+assignees: ''
+body:
+  - type: textarea
+    attributes:
+      label: Is your feature request related to a problem? Please describe.
+      description: |
+        A clear and concise description of what the problem is. Ex. I'm always frustrated when [...]
```

Placeholders might be more appropriate to communicate example values for an input field:
```suggestion
      description: |
        A clear and concise description of what the problem is.
    placeholder: e.g: I'm always frustrated when I try to [...] but [...]
        
```

---

I've moved the example in the description to a placeholder as suggested.